lib/dialects/postgres/data-types.js

```diff
@@ -87,6 +88,47 @@ module.exports = BaseTypes => {
 
   BaseTypes.DATEONLY.types.postgres = ['date'];
 
+  function INTERVAL(...args) { // TODO precision
+    if (!(this instanceof INTERVAL)) return new INTERVAL();
+    BaseTypes.INTERVAL(...args);
+  }
+
+  inherits(INTERVAL, BaseTypes.INTERVAL);
+
+  INTERVAL.parse = function parse(value) {
+    if (typeof value === 'string') {
+      return moment.duration(value);
+    }
+
+    return value;
+  };
+
+  INTERVAL.prototype._stringify = function _stringify(value) {
+    if (typeof value === 'number') {
+      return moment.duration(value).toISOString();
+    }
+
+    if (moment.isDuration(value)) {
+      return value.toISOString();
+    }
+
+    return value;
+  };
+
+  INTERVAL.prototype._sanitize = function _sanitize(value) {
+    if (typeof value === 'string') {
+      return moment.duration(value);
```

Comment:
`moment.duration` seemed like the best thing to sanitize to. Alternatively, we could normalize to strings, but then we'd potentially need to decide on a format (ISO, SQL, PG). Or we could normalize to number of milliseconds (or seconds).